src/time_travel/patchers/base_patcher.py

```diff
@@ -0,0 +1,141 @@
+"""Base class for patching time modules."""
+
+import sys
+import mock
+import inspect
+
+
+class BasePatcher(object):
+    """Base class for patching time modules."""
+    
+    def __init__(self,
+                 clock,
+                 event_pool=None,
```

Comment:
I don't think this should ever be None. If it is - it means that it's not initialized correctly by the main class. Remove the default argument pl0x.

Reply:
Done.